Documentation/release-notes/xa1025.md

```diff
@@ -0,0 +1,6 @@
+#### Application and library build and deployment
+
+- Projects using the experimental `Hybrid` value for the `AndroidAotMode`
+  MSBuild property in combination with the armeabi-v7a target ABI did not yet
+  produce a build error to help indicate that they would abort during startup
+  in armeabi-v7a ABI device environments.
```

Comment:
Should we document `$(AndroidAotModeValidateAbi)` here as well as in `BuildProcess.md`?

Reply:
Yup